<?xml version="1.0"?>

<!-- roslaunch melexis melexis_driver.launch experiment_name:="volunteer_1_1" -->



<launch>

	<arg name="port1"           	     default="/dev/ttyACM0" />

<!-- Recording Sensor1-->

	<node pkg="melexis_v2"  type="melexis_driver.py"  name="melexis_driver1" output="screen" respawn="false"    launch-prefix=""  required="true">

		<param name="port"                value="$(arg port1)"/>  
	</node>

</launch>
